batch/batch/server/server.py

```diff
@@ -829,13 +827,33 @@ def is_successful(self):
     if not validator.validate(parameters):
         abort(400, 'invalid request: {}'.format(validator.errors))
 
-    batch = await Batch.create_batch(attributes=parameters.get('attributes'),
-                                     callback=parameters.get('callback'),
-                                     userdata=userdata)
-
     job_parameters = parameters.get('jobs', [])
```

Comment:
this isn't the parameters for a single job, this is jobs_parameters or something, right? Maybe just jobs? Or jobs_dicts?